packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -112,6 +112,23 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return this.wrapResponse(res)
   }
 
+  async batchCall(address, input) {
```

Comment:
We could just simply update the `signAndSend` to accept a list of messages instead of one